tasks/__init__.py

```diff
@@ -776,7 +776,7 @@ def packages(ctx):
         'install elasticsearch',
         'install rabbitmq',
         'install node',
-        'tap tokutek/tokumx',
+        'tap chrisseto/homebrew-tokumx',
```

Comment:
Can this be pinned to a commit or anything? New policy is to not just link to an arbitrary branch or repo, especially if it's not under centerforopenscience,